libs/designsystem/src/lib/components/text-link/text-link.component.scss

```
@@ -0,0 +1,36 @@
+@import '../../scss/utils';
+@import '../icon/icon.component.shared';
+
+:host {
+  cursor: pointer;
+
+  a {
+    font-size: font-size('n');
+    line-height: line-height('n');
+    color: get-text-color('black');
+  }
+
+  kirby-icon {
+    padding-left: size('xxxs');
+    padding-right: size('s');
```

Any reason for not using kirby-icon's exposed API (custom css properties) and using margin instead? 🤔 
```suggestion
    --kirby-icon-margin-left: #{size('xxxs')};
   -kirby-icon-margin-right #{size('s')};
```

---

Can't see why not, so implemented it this way instead :)